releng_sop/koji_block_package_in_release.py

```diff
@@ -1,19 +1,37 @@
 # -*- coding: utf-8 -*-
+"""Block packages in a release.
 
+Constructs the koji command
+
+    koji --profile=KOJI_PROFILE block-pkg RELEASE_TAG PACKAGES...
+
+KOJI_PROFILE is obtained from the environment settings.
+
+RELEASE_TAG is the "tag_release" key from the release configuration
+    corresponding to the release id.
+"""
 
 from __future__ import print_function
 from __future__ import unicode_literals
 
 import argparse
-import subprocess
-
-from .common import Environment, Release
 
+from .common import Environment
 from .kojibase import KojiBase
 
 
 class KojiBlockPackageInRelease(KojiBase):
+    """Block packages in a release."""
+
     def print_details(self, commit=False):
+        """Print details of command execution.
+
+        Parameters
+        ----------
+        commit: boolean (optional; default False)
+            Flag to indicate if the command will be actually executed.
+            Line indicating "test mode" is printed, if this is False.
+        """
```

Comment:
I'd use sphinx syntax for this (:param:, :type:, ...)
See: https://github.com/release-engineering/productmd/blob/master/productmd/common.py#L81